test/e2e/tests/confirmations/header.spec.js

```diff
@@ -0,0 +1,88 @@
+const { strict: assert } = require('assert');
+const {
+  defaultGanacheOptions,
+  openDapp,
+  unlockWallet,
+  WINDOW_TITLES,
+  withFixtures,
+} = require('../../helpers');
+const FixtureBuilder = require('../../fixture-builder');
+
+describe('Header component', function () {
```

Comment:
I was also working on the signature redesign tests and did this a bit different. Not sure which way is the best but if you have a lot of nested tests you might have to keep repeating the condition everywhere. We can do the same for the it blocks. Wdyt?

```
const describeif = process.env.ENABLE_CONFIRMATION_REDESIGN
  ? describe
  : describe.skip;
```

Reply:
ok, looks like this doesn't work :(